modular_skyrat/modules/food_replicator/replicator_designs/replicator_clothing.dm

```diff
@@ -0,0 +1,39 @@
+/datum/design/utility
+	name = "utility uniform"
+	id = "slavic_util"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 200)
+	build_path = /obj/item/clothing/under/misc/skyrat/utility
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_CLOTHING)
```

Comment:
Please make all of these `category` lists into multiline ones, I don't feel like making a suggestion for each of them.

Reply:
RnD category lists aren't multiline everywhere else?..

Reply:
I know, but it's easier to read through multiline lists, especially when it's stuff with repetitive prefixes like this.

Reply:
Fair enough ig.